README.md

```
@@ -1,222 +1,124 @@
-openintel trades [--limit N] [--resolved true/false]
+## Use with an AI agent (MCP)
 
-# Export
-openintel export [--since YYYY-MM-DD] [--category CAT]
+OpenIntel can run as a local **MCP server** so an AI agent can consult its analysis while
+you trade through **Robinhood's official Agentic Trading MCP**. OpenIntel is the
+intelligence layer; the agent is the brain; Robinhood's MCP is execution.
 
-# Re-embed entries missing vectors
-openintel reindex
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)